### PR TITLE
updater: Move operators and appversions into subdirectories.

### PR DESCRIPTION
**Pull request by yifan-gu**
This helps to clean up the 'make payload' scripts.

Also fixed the tectonic-etcd-operator's appversion field.

### CODE REVIEW

File: config.tf [hunk 1]
```diff
@@ -68,12 +68,13 @@ variable "tectonic_versions" {
...
-    monitoring   = "1.3.0"
-    kubernetes   = "1.6.6+tectonic.1"
-    tectonic     = "1.6.6-tectonic.1"
+    etcd          = "3.1.8"
+    prometheus    = "v1.7.1"
+    alertmanager  = "v0.7.1"
+    monitoring    = "1.3.0"
+    kubernetes    = "1.6.6+tectonic.1"
+    tectonic      = "1.6.6-tectonic.1"
+    tectonic-etcd = "0.0.1"
```
Comment (hasbro17) [inline config.tf]: @yifan-gu Can you also change `modules/tectonic/assets.tf` to also use this variable like it's being done in `modules/update-payload/assets.tf`. It's still using the container image tag. 
https://github.com/yifan-gu/tectonic-installer/blob/f98cc0ff72eaf288d84b71c0249e65a5f87fcaef/modules/tectonic/assets.tf#L41

Comment (yifan-gu) [inline config.tf]: @hasbro17 Good catch, done.

**Comment by Quentin-M** [comment]
ok to test

**Comment by Quentin-M** [comment]
@yifan-gu: Happy to approve if it has been verified that all manifests are submitted successfully against Kubernetes. Have you deployed the cluster yourself and verified? Or should I do it?

**Comment by yifan-gu** [comment]
@Quentin-M Yeah, it's working.

**Pull request merged**